dev/AppNotifications/AppNotificationContentBuilder/AppNotificationContentBuilder.idl

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Windows.AppNotifications
+{
+    [contract(AppNotificationsContract, 2)]
+    runtimeclass ButtonConfig
+    {
+        ButtonConfig(String content);
+
```

Comment:
Should the ctor also take the action for the button? Granted it can be added using the AddArgument()n function, but users of ths API are always going to want to have an action assoiated to the button, no?

Reply:
After thinking this over a little bit more, I agree with the current API design. AddArgument() is the correct API to use to set the arguments.